word-count/word_count_test.rb

```diff
@@ -0,0 +1,74 @@
+require 'minitest/autorun'
+require_relative 'word_count'
+
+class PhraseTest < Minitest::Test
+  def test_count_one_word
+    phrase = Phrase.new('word')
+    counts = { 'word' => 1 }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.